doc/how-to/initialise.md

```diff
@@ -4,11 +4,17 @@
 The {ref}`initialisation process <explanation-initialisation>` bootstraps the MicroCloud cluster.
 You run the initialisation on one of the machines, and it configures the required services on all of the machines that have been joined.
 
+## Pre-initialisation requirements
+
+- Complete the steps in {ref}`howto-install` before initialisation.
+- If you intend to use full disk encryption (FDE) on any cluster member, that member must meet the prerequisites listed on this page before initialisation: {doc}`microceph:explanation/full-disk-encryption`. 
+  - Follow only the instructions in the Prerequisites section on that page. Skip its Usage section; the MicroCloud initialisation process handles the disk encryption.
+
 (howto-initialise-interactive)=
 ## Interactive configuration
 
 If you run the initialisation process in interactive mode (the default), you are prompted for information about your machines and how you want to set them up.
-The questions that you are asked might differ depending on your setup; for example, if you do not have the MicroOVN snap installed, you will not be prompted to configure your network, and if your machines don't have local disks, you will not be prompted to set up local storage.
+The questions that you are asked might differ depending on your setup. For example, if you do not have the MicroOVN snap installed, you will not be prompted to configure your network; if your machines don't have local disks, you will not be prompted to set up local storage.
```

Comment:
"to setup local and/or remote storage."? Both questions won't be asked if there aren't any disks available.

Reply:
Do you mean it won't ask about remote storage either if you don't have any local disks?

Reply:
Got it, I removed the word "local" so it's just "you will not be prompted to set up storage."